### PR TITLE
Dynamic groups view

### PR DESCRIPTION
**Pull request by davlin93**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -9,29 +9,94 @@
...
-
 $(function() {
   var directory = new air.File();
   var picturePath = "";
   var ratingsFile;
   var pictures = [];
+  var groups = [];
+
+  var loadPics = function() {
+    var domAdd = "";
```
Comment (kehphin) [inline js/main.js]: domAdd is ambigious, how about something like `unsortedPicsHtml`

File: js/main.js [hunk 2]
```diff
@@ -9,29 +9,94 @@
...
+
+    var groupCount = 1;
+    groups.forEach(function(group) {
+      var currentGroup = $("div[data-group='" + groupCount + "']");
+      var outHtml = "";
+
+      currentGroup.show();
+
+      var picSize;
+      if (groups.length > 2) {
```
Comment (kehphin) [inline js/main.js]: can you add a comment on what you're doing here?

**Comment by kehphin** [comment]
How did you solve the problem with pictures not getting loaded in AIR?

**Comment by davlin93** [comment]
instead of putting the raw path in the src, I used the air File `.url` helper to get the path a6a2c1bc266b7526dda311e8af6505

File: js/main.js (hunk 2 above)
Comment (davlin93) [inline js/main.js]: sure, I will. Before images were always 25% of the group width, but they seemed too tiny for 3+ groups, so I made them 50% of the width for those.

File: js/main.js [hunk 3]
```diff
@@ -9,29 +9,94 @@
...
+      domAdd += '<div class="col-md-1"><img src="' + path + '" class="image"></div>';
+    });
+
+    // flush DOM then add current pictures
+    $('#unsorted').html('');
+    $('#unsorted').append(domAdd);
+
+    var groupCount = 1;
+    groups.forEach(function(group) {
+      var currentGroup = $("div[data-group='" + groupCount + "']");
```
Comment (kehphin) [inline js/main.js:32]: Would it be cleaner here to just have each group associated by id, then fetch by id? Ex. `id='1'` then `$('#'+ groupCount)`

**Comment by kehphin** [comment]
We add the group number to the groups below the unsorted group

screen shot 2015-04-01 at 1 49 30 am

**Comment by kehphin** [comment]
Also, once the pictures are put in the groups below, I think they shouldn't be in the unsorted group. There should only be one instance of the same picture that can be moved around

File: js/main.js (hunk 3 above)
Comment (davlin93) [inline js/main.js:32]: resolved f2f

**Comment by gary2546** [comment]
+1

File: js/main.js [hunk 4]
```diff
@@ -9,29 +9,98 @@
...
+      // for smaller group boxes, scale the pictures up so they are still viewable
+      var picSize;
+      if (groups.length > 2) {
+        picSize = 6;
+      } else {
+        picSize = 3;
+      }
+
+      // add each picture inside the group to the outHtml string that will be added to DOM
+      group.forEach(function(picture) {
```
Comment (kehphin) [inline js/main.js:46]: groups

Comment (kehphin) [inline js/main.js:46]: nvm

**Comment by kehphin** [comment]
LGTM +1

**Pull request merged**